automatic/eid-belgium/tools/LICENSE.txt

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
please move this file to a `legal` directory instead, to match the convention used for most of the existing packages.

You'll also need to update the files section of course.

Reply:
Taken care of in https://github.com/wget/chocolatey-package-eid-belgium